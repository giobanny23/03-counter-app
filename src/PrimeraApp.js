import React from 'react';
import PropTypes from 'prop-types';
//import React, {Fragment} from 'react';


const PrimeraApp = ({saludo, subtitulo= 'Desarrolla Nuevas habilidades' }) => { 

   
    return (

    <>
        <h1> {saludo} </h1>
    <p>{subtitulo}</p>
    </>
    );
}
PrimeraApp.propsTypes = {
    saludo: PropTypes.string.isRequired
}
export default PrimeraApp;