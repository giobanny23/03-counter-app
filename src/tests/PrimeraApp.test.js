import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'

import PrimeraApp from '../PrimeraApp';

describe('Pruebas de PrimeraApp ', () => {
  

    test('debe de mostrar <PrimeraApp/> Correctamente', () => {
    
        
        const saludo = 'Hola soy goku';
        const wrapper = shallow(<PrimeraApp  saludo={saludo}/>)

        expect(wrapper).toMatchSnapshot();
    })
    
    
    
})
