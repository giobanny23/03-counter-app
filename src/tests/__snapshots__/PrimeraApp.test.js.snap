// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de PrimeraApp  debe de mostrar <PrimeraApp/> Correctamente 1`] = `ShallowWrapper {}`;
